analytics_data_api/v0/tests/views/test_course_summaries.py

```diff
@@ -33,7 +33,8 @@ def tearDown(self):
         self.model.objects.all().delete()
 
     def create_model(self, model_id, **kwargs):
-        for mode in kwargs['modes']:
+        modes = kwargs['modes']
```

Comment:
Could the modes key be missing from kwargs? Would `kwargs.get('modes', [])` be safer?

Reply:
That would be safer and I'll change it, though the previous implementation would have errored if 'modes' was missing